ipatests/test_integration/test_commands.py

```diff
@@ -133,3 +134,61 @@ def test_change_sysaccount_password_issue7561(self):
 
         tasks.ldappasswd_sysaccount_change(sysuser, original_passwd,
                                            new_passwd, master)
+
+
+class TestIPACAInstallCommand(IntegrationTest):
+    """
+    Test ipa-ca-install command and its options.
+    """
+    num_replicas = 1
+
+    @pytest.mark.xfail(reason='Ticket N 6985')
+    def test_install_ca_replica(self):
+        """
+        ipa-ca-install command installs CA on replica even if cert file
+        is not specified with --external-cert-file option. If executed
+        command with non-existing file, invalid file etc,
+        """
+        # related to https://pagure.io/freeipa/issue/6985
+        master = self.master
+
+        replica = self.replicas[0]
+        tasks.install_master(self.master)
+        tasks.install_replica(master, replica)
```

Comment:
Need to add the arg setup_ca=False otherwise ipa-replica-install will be called with --setup-ca